CONTRIBUTING.md

```diff
@@ -51,6 +51,12 @@ cannot be merged.
 
 ([source](http://blog.qt.io/blog/2014/06/13/qt-weekly-13-qstringliteral/))
 
+## Use .arg(arg1, arg2) instead of .arg(arg1).arg(arg2)
+
+If you just need to use a plain .arg() call, use multiple arguments within it and not as separate .arg() calls as that'll avoid creating temporary QString objects.
+
```

Comment:
Um, the fragment could make it clear that the first being a number is what triggers the expectation that the second and third would be - which is not the aim here - i.e. perhaps the second sentence should be something along the lines of:

"If your first argument happens to be a number and there are one or two further arguments in total then be sure to wrap each of them in `QString::number(yourNumber)` to force them all to be regarded as strings {this is to avoid the two and three argument cases of .arg() where a numeric first argument is converted to a string with a width and base as specified by the other arguments}."

Although it might be possible to put that in less words... 😉